cmd/cli/options/convert.go

```diff
@@ -0,0 +1,19 @@
+package options
+
+import (
+	"github.com/spf13/cobra"
+)
+
+// ConvertOptions defines the options for the `convert` command
+type ConvertOptions struct {
+	Format     string
+	Encoding   string
+	OutputPath string
+}
+
+// AddFlags adds command line flags for the ConvertOptions struct
+func (o *ConvertOptions) AddFlags(cmd *cobra.Command) {
+	cmd.Flags().StringVarP(&o.Format, "format", "f", "", "format string")
+	cmd.Flags().StringVarP(&o.Encoding, "encoding", "e", "", "output encoding")
+	cmd.Flags().StringVarP(&o.OutputPath, "output", "o", "", "path to write the converted SBOM")
```

Comment:
i am a bit confused about the `--encoding` and `--format` flags ,
i would love to allow the user to only change the version similar to the encoding one.

Reply:
I took the "encoding" idea from your PR. encoding being [json, text, ... ] when format being [spdx, cyclonedx, cyclonedx-1.4 ... ]

I would prefer it way more than going with spdx-json, spdx-tv

@bardenstein what do you think?

Reply:
this is a bit different from other SBOM generators, that have things like --format=spdx-json or --format=cyclonedx-xml. 
But I'm fine with splitting them, having --encoding optional and defaulted to json.